reference/checkouts.sf.yml

```diff
@@ -3762,11 +3762,11 @@ definitions:
                   variantId:
                     description: ID of the variant.
                     type: number
-                    format: double
+                    format: integer
```

Comment:
Rather than leaving the type as `number` and changing `format` to `integer`, these should instead be `type: integer` since `number` and `integer` are two distinct data types: https://swagger.io/docs/specification/data-models/data-types/

``` yaml
                  variantId:
                    description: ID of the variant.
                    type: integer
```